README.md

```diff
@@ -118,7 +118,7 @@ $number->formatForCallingFrom('US'); // 011 44 7123 456789
 ### Number types
 
 In certain cases, it is possible to know the type of a phone number (fixed line, mobile phone, etc.), using
-the `getNumberType()` method, which returns a constant from the [PhoneNumberType](https://github.com/brick/phonenumber/blob/0.2.0/src/PhoneNumberType.php) class:
+the `getNumberType()` method, which returns a constant from the [PhoneNumberType](https://github.com/brick/phonenumber/blob/0.4.0/src/PhoneNumberType.php) class:
```

Comment:
Here again. Use master instead of version:
https://github.com/brick/phonenumber/blob/master/src/PhoneNumberFormat.php